release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -101,28 +105,28 @@ A guiding principle for the lead is be an arbiter of decisions, and not the prim
 <p>- Create the retrospective document and corresponding bit.ly link
-<p>- Identify vacant release team roles and begin asking SIGs, the community, and CNCF-sponsor companies for volunteers
-<p>- Begin meeting with SIGs to introduce yourself</td>
-<p>- Request review of this document by the release lead shadow.  The shadow should also take all actions in this document around joining groups and requesting access permissions.
+<p>- Begin meeting with SIGs to introduce yourself
+<p>- Begin paying attention to CI signal, as it may begin degrading soon after the prior release is cut and any slips must be caught and rectified promptly.
+<p>- Request review of this document by the release lead shadow.  The shadow should also take all actions in this document around joining groups and requesting access permissions.</td>
   </tr>
   <tr>
     <td>2</td>
     <td><p>- Assist the features lead in collecting planned work from SIGs
 <p>- Schedule weekly release team meetings at 10 am Pacific time on a day that is most acceptable to the team.  These will eventually turn into burndown meetings and occur daily.  Invite the kubernetes-milestone-burndown@googlegroups.com group.  You will need to contact SIG-Release leads to gain access to the SIG's zoom.us account for hosting/recording/posting meeting video.
+<p>- Poll release team membership and schedule a weekly alternate meeting to better enable more attendance outside of the Americas.
 <p>- Create a Google Calendar for the release:  Follow the steps in <a href="https://git.k8s.io/community/sig-governance.md">the sig-governance documentation</a> to create and share a calendar with the community.  Add key event dates during the cycle.
 <p>- Begin reporting release status at the community meeting
-<p>- Ensure the release team is fully filled if possible, with members subscribed to the kubernetes-milestone-burndown@googlegroups.com group.
-<p>- Begin paying attention to CI signal, as it will begin degrading soon after the prior release is cut
 <p>- Continue meeting with SIGs for introductions</td>
   </tr>
   <tr>
     <td>3</td>
     <td><p>- Create the release notes draft file in the release directory per the standard above
-<p>- Prepare for x.x.0-alpha.2 release, specifically that there is a branch manager on the team, and that master-blocking tests are all green
+<p>- Prepare for x.x.0-alpha.0 "release", specifically that there is a branch manager on the team, and that master-blocking tests are all green.  The alpha.0 artifacts were created already as a part of the prior release.  But this synthetic notation is a point to review process with the branch manager.  Request access to GCB through Caleb Miles for branch manager lead and optionally also read-only access (if system supports this) for the release lead and release lead shadow.
```

Comment:
> Request access to GCB through Caleb Miles

It's worth mentioning how to request and access through Caleb Miles (contacts, etc.).